thali/thalicryptomanager.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+var path = require('path');
+
+var pkcs12FileName = '/pkcs12.pfx';
+var password = 'password';
```

Comment:
Please put in a comment specifying that we use a bogus password because using a password at all is bogus. Anyone who can get to the file can get to the app and thus can get the password. So we just include something because the API expects it. If and when mobile devices start having secure key stores that are tagged to app IDs then we can discuss using a real password.